src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -7717,7 +7717,7 @@ void MusicXmlParserPass2::harmony(const String& partId, Measure* measure, const
 
     FretDiagram* fd = nullptr;
     Harmony* ha = Factory::createHarmony(m_score->dummy()->segment());
-    HarmonyInfo* info = new HarmonyInfo(m_score);
+    HarmonyInfo* info = nullptr;
```

Comment:
Can we keep the initialisation here? Instead of `if (info)` below we should check `info->rootTpc() != Tpc::TPC_INVALID`. If for any reason `root` isn't the first tag, this will cause crashes below.

Reply:
Good point, that actually simplifys it a lot. Thanks!